sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AdaptiveRulesHolder.scala

```diff
@@ -26,5 +26,6 @@ import org.apache.spark.sql.execution.SparkPlan
  */
 class AdaptiveRulesHolder(
     val queryStagePrepRules: Seq[Rule[SparkPlan]],
-    val runtimeOptimizerRules: Seq[Rule[LogicalPlan]]) {
+    val runtimeOptimizerRules: Seq[Rule[LogicalPlan]],
```

Comment:
shall we rename it? it's hard to tell the difference between this and `queryStageOptimizerRules`

Reply:
nvm, this matches the existing API in `SparkSessionExtensions`

Reply:
can we add some comments to explain the difference?

Reply:
addressed